test/e2e/config/azure-dev.yaml

```diff
@@ -115,7 +115,7 @@ variables:
 intervals:
   default/wait-controllers: ["3m", "10s"]
   default/wait-cluster: ["20m", "10s"]
-  default/wait-control-plane: ["20m", "10s"]
+  default/wait-control-plane: ["35m", "10s"]
```

Comment:
I don't think we want to change this value for every test, was it to address #1481 ? if so we should fix it separately and/or add a specific AKS timeout interval

Reply:
Not sure if this is the only problem. But I have noticed several times for an AKS cluster to come up fully it takes a max of 30m I kept 5m buffer.

Reply:
were there any other changes to make e2e pass? I noticed deletion and the full test sometimes takes more than 20min, but the cluster initialization should really take less time. it's fine if we want to debug it elsewhere, but let's be cognizant of that.

I'd probably prefer adding a separate interval for AKS if we need to increase it so high, like cecile mentioned.

Reply:
Nvm, answered my own question in the other issue/pr

Reply:
let’s revert the change here, I’ll fix the timeout

Reply:
No, i did not make any other changes.  Yes it gets initialised quickly but for the first managedmachinepool or first system managed machine pool to come to a **ready** state it takes time. Like about 25-30 mins approx. I have not exactly timed the deletion but from the boot to until the first system pool comes up it would be approx 30 mins.

Reply:
I debugged this extensively in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1488, there's a lag in Azure List VMSS which causes AMMP reconciliation to take way longer than expected. Both pools and the cluster should generally be done in ~10 minutes. I have a workaround and I'm also chasing it down internally with the owning team.

There's also a small issue with watches where if we reconcile AMMP with a stale AMCP in the cache, we won't requeue the 2nd machinepool for a long time (because our mapper only maps to system pools).